pages/index.tsx

```diff
@@ -46,47 +47,64 @@ export default function Home(): JSX.Element {
 
   return (
     <>
-      {/* <Navbar /> */}
+      <Navbar />
       <Page className="homepage-container">
         <Text h2 style={{ marginBottom: '5px' }}>
           Mercury
         </Text>
         <Text className="sub-heading">{orgName} Email System</Text>
-        {user?.isAdmin && <>you&apos;re an admin</>}
-
         <br />
-        <Divider align="start">E-mail Recipients</Divider>
-        <Text className="sub-heading">Choose which lists to send a mail to:</Text>
-        <Checkbox.Group value={[]} size="medium" onChange={mailingListSelectionHandler}>
-          {mailingLists.map((mList, i) => (
-            <Checkbox key={`mailing-list-${i}`} value={mList.address}>
-              {mList.name}
-            </Checkbox>
-          ))}
-        </Checkbox.Group>
+        {/* If user is not admin, deny access and prompt them to login */}
+        {!user?.isAdmin && (
```

Comment:
You could switch to using a conditional statement:

line 58: `user?.isAdmin ?`
line 59-60: `"Access Denied"`
line 70-72: `"Email Recipients"`

Reply:
I was not thinking straight when i write that code, ty

Reply:
Made the changes :+1: